src/main/java/com/capitalone/dashboard/model/WhiteSourceRefreshRequest.java

```diff
@@ -4,26 +4,15 @@
 
 public class WhiteSourceRefreshRequest {
 
-    @ApiModelProperty(notes = "WhiteSource Organization name",name="OrgName",required=true)
```

Comment:
let's not delete orgName and productName for now, lets keep them required false.

Reply:
👍 placed them back in the request obj